source/js/buyers-guide/homepage-c-slider.js

```diff
@@ -118,21 +124,39 @@ export default {
       }
     );
 
+    /**
+     * querySelectors in this event listener are constantly polled on scroll
+     * to ensure when a browser width changes creepo-face still works as expected
+     * And getElementById is used in places to speed up DOM searching
+     */
     window.addEventListener(
       "scroll",
       () => {
-        if (window.innerWidth < 768) {
-          const element = document.getElementById("product-list");
-          const position = element.getBoundingClientRect();
-          /**
-           * Check if the product grid area is partially visible in the viewport
-           */
-          if (position.top < window.innerHeight && position.bottom >= 0) {
-            STICKY_BAR.classList.add("search-active");
-          } else {
-            // Product area is no longer in the viewport at all.
-            STICKY_BAR.classList.remove("search-active");
-          }
+        const element = document.getElementById("product-list");
```

Comment:
Let's check if `document.getElementById("product-list");` exist before we do anything more to it. I would give it a more explicit var name since we are working with many DOM nodes in this event handler.